airflow/hooks/S3_hook.py

```diff
@@ -275,7 +275,8 @@ def load_file(self,
                   key,
                   bucket_name=None,
                   replace=False,
-                  encrypt=False):
+                  encrypt=False,
+                  upload_args={}):
```

Comment:
nit: we shouldn't use {} as default value(https://docs.quantifiedcode.com/python-anti-patterns/correctness/mutable_default_value_as_argument.html) . Should just do ```upload_args=None```

Reply:
This can be quite dangerous indeed if you're not familiar with its behaviour: 
```
MacBook-Pro-van-Fokko:~ fokkodriesprong$ python3
Python 3.7.0 (default, Jun 29 2018, 20:13:13) 
[Clang 9.1.0 (clang-902.0.39.2)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> def some_function(a={}):
...     return a
... 
>>> some_function()
{}
>>> some_function()['lid'] = True
>>> some_function()
{'lid': True}
```